plugins/rest-plugin/src/main/java/ru/sbtqa/tag/api/manager/EndpointManager.java

```diff
@@ -20,9 +20,10 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(EndpointManager.class);
     private static final ApiConfiguration PROPERTIES = ConfigFactory.create(ApiConfiguration.class);
-    private static final Set<Class<?>> ENDPOINTS_CACHE = new HashSet<>();
+    private static final ThreadLocal<Set<Class<?>>> ENDPOINTS_CACHE = ThreadLocal.withInitial(() -> new HashSet<Class<?>>());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this lambda with a method reference. ([squid:S1612](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1612))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=126&issues=AWeIHODYcrQ7oSxgxnf9&open=AWeIHODYcrQ7oSxgxnf9)

[ISSUE_KEY]: AWeIHODYcrQ7oSxgxnf9